modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/forms/input/InputLocalized.scss

```diff
@@ -1,4 +1,4 @@
-.lfr-objects__input-localized {
+.input-localized {
```

Comment:
The implementation of this wrapper hides by default the text under the input. We could consider make it customizable with a prop and let hidden by default to avoid breaking the usage in Objects & Notifications module.

![image](https://github.com/liferay-frontend/liferay-portal/assets/132653342/98ff5de6-2242-4698-bf37-a8d2e34e6d45)

Reply:
Without any of the context, I'd say that's a good idea. Avoiding breaking changes is always good 😄

Reply:
Found in Clay docs that [resultFormatter](https://clayui.com/docs/components/localized-input/api.html#api-resultFormatter) and [helpText](https://clayui.com/docs/components/localized-input/api.html#api-helpText) use a `.form-text` div to display its  contents.

To prevent loosing functionality and keep current behavior in actual usages I'm passing a `resultFormatter` with a function that returns `null` as the default value.